### PR TITLE
OSD-20509: Updates infra resize command to add SL for GCP

### PR DESCRIPTION
**Pull request by PriyankaKanthale94**
for [OSD-20509](https://issues.redhat.com//browse/OSD-20509)

### CODE REVIEW

File: cmd/cluster/resize/infra_node.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package resize
 
 // cspell:ignore embiggen
-
+/
```
Comment (devppratik) [inline cmd/cluster/resize/infra_node.go:4]:
```suggestion

```

File: cmd/cluster/resize/infra_node.go [hunk 2]
```diff
@@ -283,8 +283,8 @@ func (r *Resize) RunInfra(ctx context.Context) error {
 	}
 
 	// Delete temp machinepool
-	log.Printf("deleting temporary machinepool %s, with instance type %s", tempMp.Name, instanceType)
-	if err := r.hiveAdmin.Delete(ctx, tempMp); err != nil {
+	log.Printf("deleting temporary machinepool %s, with instance type %s", .Name, instanceType)
+	if err := r.hiveAdmin.Delete(ctx, ); err != nil {
```
Comment (shitaljante) [inline cmd/cluster/resize/infra_node.go:287]: please keep these functions as it is.

**Comment by devppratik** [comment]
/retitle [OSD-20509] Updates infra resize command to add SL for GCP

File: cmd/cluster/resize/infra_node.go [hunk 3]
```diff
@@ -27,11 +26,12 @@ import (
...
 const (
-	twentyMinuteTimeout                = 20 * time.Minute
-	twentySecondIncrement              = 20 * time.Second
-	resizedInfraNodeServiceLogTemplate = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized_auto.json"
-	infraNodeLabel                     = "node-role.kubernetes.io/infra"
-	temporaryInfraNodeLabel            = "osdctl.openshift.io/infra-resize-temporary-machinepool"
+	twentyMinuteTimeout                   = 20 * time.Minute
+	twentySecondIncrement                 = 20 * time.Second
+	resizedInfraNodeServiceLogTemplate    = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized_auto.json"
+	GCPresizedInfraNodeServiceLogTemplate = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/gcp/GCP_infranode_resized_auto.json"
```
Comment (mjlshen) [inline cmd/cluster/resize/infra_node.go:32]: I don't think the service log listed here as "GCP" is any better - the documentation it links to still only talks about ROSA

**Review by mjlshen — CHANGES_REQUESTED** [review]
/label tide/merge-method-squash

File: cmd/cluster/resize/infra_node.go (hunk 3 above)
Comment (PriyankaKanthale94) [reply cmd/cluster/resize/infra_node.go:32]: @mjlshen GCP SL points to OSD documentation not the ROSA documentation. We do not have a separate documentation for GCP on OSD.  AWS on OSD and GCP on OSD are in single documentation.

**Pull request merged**